cli/lib/kontena/client.rb

```diff
@@ -55,6 +55,7 @@ def initialize(api_url, token = nil, options = {})
         connect_timeout: ENV["EXCON_CONNECT_TIMEOUT"] ? ENV["EXCON_CONNECT_TIMEOUT"].to_i : 5,
         read_timeout:    ENV["EXCON_READ_TIMEOUT"]    ? ENV["EXCON_READ_TIMEOUT"].to_i    : 30,
         write_timeout:   ENV["EXCON_WRITE_TIMEOUT"]   ? ENV["EXCON_WRITE_TIMEOUT"].to_i   : 5,
+        ssl_ca_path:     ENV["EXCON_SSL_CA_PATH"]     ? ENV["EXCON_SSL_CA_PATH"] : "/etc/ssl/certs",
```

Comment:
I'm not sure it's a good idea to default to `/etc/ssl/certs` without the ENV.

Excon will already try using `ENV['SSL_CERT_FILE']` or `ENV['SSL_CERT_DIR']` if this option is not set.

https://github.com/excon/excon/blob/master/lib/excon/ssl_socket.rb#L34